src/components/Users/User/User.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useState} from 'react';
+import Button from 'components/core/Button/Button';
+import Input from 'components/core/Input/Input';
+import UserModel from 'model/UserModel';
+import classes from './User.module.scss';
+
+interface UserProps {
+  user : UserModel;
+  editUser: (id: string | number, user: UserModel) => void;
+  deleteUser: (id: string | number) => void;
+}
+
+const User: React.FC<UserProps> = ({user, editUser, deleteUser}) => {
```

Comment:
Can we dest object agin, exp. { user :{ title, startRentDate, endRentDate }, editUser, deleteUser }